src/main/java/org/codeforamerica/messaging/services/MessageService.java

```diff
@@ -104,10 +104,8 @@ public void sendMessage(Long messageId) {
                 SmsMessage sentSmsMessage = this.smsService.sendSmsMessage(message.getToPhone(), smsBody);
                 message.setSmsMessage(sentSmsMessage);
                 messageRepository.save(message);
-            } catch (IOException e) {
-                log.error("Error processing SMS template: " + templateVariant.getId());
             } catch (Exception e) {
-                log.error("Error sending SMS job", e);
+                log.error("Error sending SMS job, templateId=" + templateVariant.getId(), e);
```

Comment:
Minor: IMO, it would look better if we parameterized the message as described in https://www.slf4j.org/faq.html#paramException.